tools/c7n_azure/tests/test_app_service_plan.py

```diff
@@ -127,8 +132,7 @@ def test_resize_plan_from_resource_tag(self, update_mock):
                      'type': 'resource',
                      'key': 'tags.sku'
                  }}],
-        })
-
+        }, validate=True)
```

Comment:
We don't need validation here

Reply:
Why not validate here too?

Reply:
validation is expensive. It adds ~0.4 seconds to each test run (0.45s vs 0.05s). Purpose of this test is not schema validation, so assumption is that you provide proper policy. (I think this is the reason validation is False by default for `_load_policy` helper)